eng/automation/GenerateAndBuildLib.ps1

```diff
@@ -0,0 +1,106 @@
+#Requires -Version 7.0
```

Comment:
We have all the scripts inside the scripts folder [here](https://github.com/Azure/azure-sdk-for-net/tree/main/eng/scripts). Can we move the automation folder or all the scripts [here](https://github.com/Azure/azure-sdk-for-net/tree/main/eng/scripts)?

Reply:
update